libpod/runtime_ctr.go

```diff
@@ -431,21 +431,15 @@ func (r *Runtime) removeContainer(ctx context.Context, c *Container, force bool,
 		// from the state elsewhere
 		if !removePod {
 			if err := r.state.RemoveContainerFromPod(pod, c); err != nil {
-				if cleanupErr == nil {
-					cleanupErr = err
-				} else {
-					logrus.Errorf("removing container from pod: %v", err)
-				}
+				cleanupErr = err
```

Comment:
This changes the code flow here - will cause double-printing of errors

Reply:
reverting